.readthedocs.yml

```diff
@@ -10,6 +10,10 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.11"
+  apt_packages:
+    - ghostscript
+    - graphviz
```

Comment:
Thanks for adding this!

Turns out that I need to merge my PR to get a fix in :)
https://github.com/LLNL/axom/pull/1171/commits/a46480514e4ec886a94351bda1b27813cbd500f6